sqlglot/generator.py

```diff
@@ -647,9 +647,11 @@ def sep(self, sep: str = " ") -> str:
     def seg(self, sql: str, sep: str = " ") -> str:
         return f"{self.sep(sep)}{sql}"
 
-    def pad_comment(self, comment: str) -> str:
-        comment = " " + comment if comment[0].strip() else comment
-        comment = comment + " " if comment[-1].strip() else comment
+    @staticmethod
+    def pad_comment(comment: str) -> str:
```

Comment:
Nit: I understand this was changed because `self` is not referenced in the body of the method, but note that we don't use `staticmethod`s anywhere else in the code base at the moment, so this creates a small inconsistency.

@tobymao wdyt? Should we refactor other similar methods into `@staticmethod`s?